src/vm/dllimport.cpp

```diff
@@ -6078,9 +6096,13 @@ static void DetermineLibNameVariations(const char* const** libNameVariations, in
             static const char* const SuffixFirst[] =
             {
                 "%.0s%s%s", // name+suffix
```

Comment:
For name without any dots, this is going to probe twice now:

- First time, we append the `.dll`
- Second time, we pass in name without `.dll` suffix and LoadLibrary appends it internally.

It would be nice to avoid this duplication (e.g. just let LoadLibrary take care of appending .dll to the simple names).

Reply:
Fixed